Makefile

```diff
@@ -23,9 +23,9 @@ endif
 ## Install Python Dependencies
 requirements:
 ifeq (${CPU_OR_GPU}, gpu)
-	conda install cudatoolkit=11.0.3 cudnn=8.0 -c conda-forge
+	conda install -y cudatoolkit=11.0.3 cudnn=8.0 -c conda-forge
 endif
-	$(PYTHON_INTERPRETER) -m pip install -U pip setuptools wheel torch
+	$(PYTHON_INTERPRETER) -m pip install -U pip torch
```

Comment:
have you confirmed that removing setuptools and wheel here doesn't break anything (e.g. that having it in pyproject.toml alone seems sufficient)? for example, if i pip install from the github repo and then try `import zamba`, does that work?

Reply:
Yes, they are already installed when the environment is created. Removing them here prevents them from being updated.

This might not be strictly necessary because the version of setuptools installed by pip appears *not* to be the version used to make the editable installation of zamba during the initial setup. However, I am reluctant to leave the installation with a version of setuptools that could break things down the road.